packages/ra-example/app.js

```diff
@@ -21,13 +19,9 @@ import {
 import { UserList, UserEdit, UserCreate, UserIcon, UserShow } from './users';
 
 import data from './data';
-import * as customMessages from './i18n';
 import authClient from './authClient';
 
-const messages = {
-    fr: { ...frenchMessages, ...customMessages.fr },
-    en: { ...englishMessages, ...customMessages.en },
-};
+import englishMessages from './i18n/en';
```

Comment:
you probably forgot to update the English messages to use the custom messages, like in French.